hang_man/main.js

```diff
@@ -0,0 +1,114 @@
+const mystery_words_array = [
```

Comment:
this is going to be painful when switching between JS and Ruby, but it's customary for variables in JS to be camelCase :camel: and in Ruby snake_case :snake: